server/container_create_linux.go

```diff
@@ -871,6 +871,19 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 		return nil, err
 	}
 
+	runtimeType, err := s.Runtime().ContainerRuntimeType(ociContainer)
+	if err != nil {
+		return nil, err
+	}
+	// If using kata runtime, the process label should be set to container_kvm_t
+	// Note: the requirement here is that the name used for the runtime class has "kata" in it
+	if runtimeType == libconfig.RuntimeTypeVM && strings.Contains(strings.ToLower(sb.RuntimeHandler()), "kata") {
```

Comment:
I know kata v1 is being deprecated, but right now, there is a RuntimeTypeOCI that is technically a kata type.

We could change this to an `||` instead of an `&&`, so we always set KVM label if we're runtime VM, and then use the kata keyword if we're OCI but using kata as an OCI type.